src/rebuild/scan.c

```diff
@@ -745,8 +827,14 @@ rebuild_scanner(void *data)
 	param.ip_flags = VOS_IT_FOR_REBUILD;
 	xarg.arg = arg;
 	xarg.yield_freq = DEFAULT_YIELD_FREQ;
-	rc = vos_iterate(&param, VOS_ITER_COUUID, false, &anchor,
+
+	if (rebuild_status_match(rpt, PO_COMP_ST_UP)) {
+		rc = vos_iterate(&param, VOS_ITER_COUUID, false, &anchor,
+			 reint_cont_destroy_cb, NULL, &xarg);
```

Comment:
If the destroy is only for testing purpose temporarily, then probably it should not be in rebuild code here. Probably you can twist the current pool destroy path, i.e. only destroy the specific target instead of destroy the pool on all target.